tests/kafkatest/tests/tools/kibosh_test.py

```diff
@@ -70,12 +70,16 @@ def test_kibosh_service(self):
                 [self.nodes[0].name], KiboshTest.TARGET, "/foo", 12)
         node = self.nodes[0]
 
-        def check(self, node):
+        def check(self, node, expected_json):
             fault_json = self.kibosh.get_fault_json(node)
-            expected_json = json.dumps({"faults": [spec.kibosh_message]})
             self.logger.info("Read back: [%s].  Expected: [%s]." % (fault_json, expected_json))
             return fault_json == expected_json
 
+        wait_until(lambda: check(self, node, '{"faults":[]}'),
```

Comment:
Where do we initialize the `{"faults":[]}` array in the code?

Reply:
This is the default value when nothing is present.